agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryItemExporter.java

```diff
@@ -116,6 +132,15 @@ public CompletableResultCode shutdown() {
   CompletableResultCode internalSendByConnectionString(
       List<TelemetryItem> telemetryItems, String connectionString) {
     List<ByteBuffer> byteBuffers;
+
+    // Don't send _OTELRESOURCE_ custom metric when OTEL_RESOURCE_ATTRIBUTES env var is empty
+    if (!attributes.isEmpty()) {
+      telemetryItems.add(
+          createOtelResourceMetric(
+              connectionString,
+              telemetryItems.get(0).getTags(),
+              getProperties(telemetryItems.get(0))));
```

Comment:
do we need to add tags and properties from `telemetryItems.get(0)` to the otel resource metric?

Reply:
it's a custom metric.. should we apply common tags and properties? maybe that's a question for Ingestion team. does customer have access to resources attributes table? if they do, i would assume they want to the same tags and properties.